invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoanDetails/components/LoanMetadata/LoanMetadata.js

```diff
@@ -19,6 +27,57 @@ export default class LoanMetadata extends Component {
     this.setState({ isAvailableItemsVisible: newState });
   };
 
+  _renderItem() {
+    const data = this.props.loanDetails;
+    return (
+      !_isEmpty(data.metadata.item) && (
```

Comment:
I think this happening only when the loan is in pending state right? In that case I would say is better to check the condition that semantically is correct. WDYT?

Reply:
For the first question, I am under the impression that only documents can have loans on pending state. For the second one, I didn't understand what you are suggesting :/

Reply:
You are right about the pending loans! But why do we need to check if `item` ref  is empty?

Reply:
> For the first question, I am under the impression that only documents can have loans on pending state. For the second one, I didn't understand what you are suggesting :/

I wouldn't given for granted: you could have systems that allow both.
@zzacharo the check if in case the Item is deleted.
@topless maybe a comment or a named function could help for above and below code?
pseudo-code:
```
function isItemDeleted() {
...
}

return(!isItemDeleted(...))
```